### ISSUE TITLE
config parameter does not work

### ISSUE THREAD

**Issue by torvitas** [OpenPoGo/OpenPoGoBot#317]
### Expected Behavior

```
python pokecli.py foo/bar/config.yml
```

Loads config from given path if available otherwise throws error complaining about none existent file.
### Actual Behavior

```
python pokecli.py foo/bar/config.yml
```

Throws some out of range index error:

```
pogobot_1  | Traceback (most recent call last):
pogobot_1  |   File "pokecli.py", line 93, in <module>
pogobot_1  |     main()
pogobot_1  |   File "pokecli.py", line 72, in main
pogobot_1  |     config = init_config()
pogobot_1  |   File "pokecli.py", line 54, in init_config
pogobot_1  |     config_path = sys.argv[2]
pogobot_1  | IndexError: list index out of range
```
### Steps to Reproduce

run

```
python pokecli.py foo/bar/config.yml
```
### Other Information

OS: CentOS
Git Commit: 039613290cd859cd88ee822b8da181b4aba5efd6

**Comment by wchill** [comment]
Please post the stack trace here.

**Comment by torvitas** [comment]
preparing pull request..

**Comment by andythorne** [comment]
Fixed in #270
